.eslintignore

```diff
@@ -8,3 +8,5 @@ views
 # Autogenerated files
 lib/handlers/asm-docs-amd64.js
 lib/handlers/asm-docs-arm32.js
+lib/handlers/asm-docs-java.js
+etc/scripts/vendor/jvms.html
```

Comment:
This is handy for whenever the script has been invoked, otherwise you'll get bombarded by eslint errors for the downloaded html file